### PR TITLE
Direction step view

### PR DESCRIPTION
**Pull request by hilmyveradin**
## Description
Make the sheet view for start and finish 

## Related Issues
#40 

## Media
image

## Test Instructions

1. Click start when when adding origin/destination view
2. See if the sheets pops up
3. Try to click one of the elements there. See if it's autoscrolled and zoomed in the map points

### CODE REVIEW

File: OTPKit/Features/TripPlanner/Direction/DireactionLegOriginDestinationView.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
+//
+//  DireactionLegOriginDestinationView.swift
+//  OTPKit
+//
+//  Created by Hilmy Veradin on 08/08/24.
+//
+
+import SwiftUI
+
+struct DireactionLegOriginDestinationView: View {
```
Comment (aaronbrethorst) [inline OTPKit/Features/TripPlanner/Direction/DireactionLegOriginDestinationView.swift:10]: please rename to `DirectionLegOriginDestinationView` - you have an extra `a` in the first word :)

**Review by aaronbrethorst — COMMENTED** [review]
head's up on a typo

File: OTPKit/Features/TripPlanner/Direction/DireactionLegOriginDestinationView.swift (hunk 1 above)
Comment (hilmyveradin) [reply OTPKit/Features/TripPlanner/Direction/DireactionLegOriginDestinationView.swift:10]: Whoops. My bad. Thanks for pointing that out ^^ @aaronbrethorst

**Review by aaronbrethorst — APPROVED** [review]
great work! 👍

**Pull request merged**